.github/workflows/cd.yml

```diff
@@ -21,24 +21,25 @@ jobs:
       - name: build zero-dependency docs
         uses: ./actions/redoc
 
-      - name: commit & push bundle, docs to branch
+      - name: build postman collection
+        uses: ./actions/contract_tests
+        with:
+          testCommand: "npm run postman"
+        env:
+          LOB_API_TEST_TOKEN: ${{ secrets.LOB_API_TEST_TOKEN }}
+
+      - name: commit & push bundle, collection and docs to branch
         shell: bash
         run: |
+          set -x
           git update-index -q --refresh
           changes=$(git status --porcelain)
           if [[ "${changes}" ]]; then
             git config user.name lobot
             git config user.email lobot@lob.com
+            git add --force dist/Lob-API-postman.json
             git add --force dist/Lob-API-bundled.yml
             git add --force docs/index.html
             git commit -m "update bundled spec, zero-dependency docs"
```

Comment:
Interesting that we're doing this generation after push and not in e.g. a pre-commit hook that e.g. gets verified as matching the build on push. One the one hand, this seems like it will pollute the git log; on the other hand -- it does rather neatly solve any issues with potential conflicts in the generated files being merged.

Reply:
Well, it isn't perfect by any means. Ultimately, I don't mind the extra commits as they fade into the background pretty much instantly. Hmmm... I think I'll update the commit message to say "CD: update bundled spec, zero-dep docs, postman collection", which should fade even more effectively. As a trade-off for totally reliable updates, felt like a no-brainer to me.